packages/puppeteer-core/src/bidi/Frame.ts

```diff
@@ -148,30 +134,18 @@ export class BidiFrame extends Frame {
       timeout = this.#timeoutSettings.navigationTimeout(),
     } = options;
 
-    const readinessState = lifeCycleToReadinessState.get(
-      getWaitUntilSingle(waitUntil)
-    ) as Bidi.BrowsingContext.ReadinessState;
+    const [wait, network] = getBiDiReadynessState(waitUntil);
```

Comment:
```suggestion
    const [waitEvent, network] = getBiDiReadynessState(waitUntil);
```

Reply:
I think both are not correct it should be `readyness` or `readynessState` as it's not an event.